arch/loongarch/chunkset_lasx.c

```diff
@@ -0,0 +1,127 @@
+/* chunkset_lasx.c -- LASX inline functions to copy small data chunks based on AVX2.
```

Comment:
Might want to add comma before "based on AVX2.". Would be more readable to say "based on AVX2 version."